shared/resourcemapping/src/main/java/com/google/cloud/opentelemetry/resource/ResourceTranslator.java

```diff
@@ -172,7 +172,7 @@ public static AttributeMapping create(
   public static GcpResource mapResource(Resource resource) {
     String platform = resource.getAttribute(ResourceAttributes.CLOUD_PLATFORM);
     if (platform == null) {
-      return genericTaskOrNode(resource);
+      return mapK8sResourceOrGenericTaskOrNode(resource);
```

Comment:
Could you explain your reasoning behind this? 

If platform is `null`, we do not want it mapped to K8s.